bridge/bridge.go

```diff
@@ -244,6 +251,7 @@ func (b *Bridge) Open() (err error) {
 		return errors.Wrap(err, "can't open discord")
 	}
 
+	b.ircListener.Password = b.Config.IRCServerPass
```

Comment:
You probably want to move this here https://github.com/qaisjp/go-discord-irc/blob/830ac77/bridge/bridge.go#L272

that way it will work for all connections